### ISSUE TITLE
Implement assertIsAssocArray

### ISSUE THREAD

**Issue by peter279k** [adhocore/asserts#7]
## Introduction

**This is a**

- [X] Feature request

As title, I think we can implement this method.

And the approach is as follows:

```php
....
public function assertIsAssocArray($array)
{
    return \array_keys($array) !== \range(0, \count($array) - 1);
}
....
```

**Comment by adhocore** [comment]
yes great. go ahead with PR 😎

**Comment by peter279k** [comment]
@adhocore, thank you for your reply.

I will create the related PR for this issue :sunglasses:.

**Comment by peter279k** [comment]
Closed via PR #8.
